toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -466,34 +472,80 @@ func handleBootLoader(baseConfigPath string, config *imagecustomizerapi.Config,
 	return nil
 }
 
+// Inserts new kernel command-line args into the grub config file.
+func addKernelCommandLine(kernelExtraArguments imagecustomizerapi.KernelExtraArguments,
+	imageChroot *safechroot.Chroot,
+) error {
+	var err error
+
+	extraCommandLine := strings.TrimSpace(string(kernelExtraArguments))
```

Comment:
nit: this can be part of `AddKernelCommandLine()`

Reply:
I also added to `AddKernelCommandLine`. I kept this one as it is useful to avoid the `"Setting KernelCommandLine.ExtraCommandLine"` log message.